packages/prisma-client/prisma/migrations/20221021172622_asset_format_notnull/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "Asset" ALTER COLUMN "format" SET NOT NULL;
```

Comment:
not sure why this is here

Reply:
Seems like someone have changed `schema.prisma` but didn't create a migration. I had to add this migration before I could add mine.